services/issue/label.go

```diff
@@ -61,3 +61,13 @@ func RemoveLabel(issue *models.Issue, doer *models.User, label *models.Label) er
 	notification.NotifyIssueChangeLabels(doer, issue, nil, []*models.Label{label})
 	return nil
 }
+
+// ReplaceLabels removes all current labels and add new labels to the issue.
+func ReplaceLabels(issue *models.Issue, doer *models.User, labels []*models.Label) error {
+	if err := issue.ReplaceLabels(labels, doer); err != nil {
+		return err
+	}
+
+	notification.NotifyIssueChangeLabels(doer, issue, labels, nil)
```

Comment:
Maybe we should be notifying on deleted labels as well, and refrain to include unchanged labels.

Reply:
@guillep2k done